JRC Publications Repository.js

```diff
@@ -0,0 +1,477 @@
+						"firstName": "Tom",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Gasparri",
+						"firstName": "Ignacio",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Munger",
+						"firstName": "Jacob",
+						"creatorType": "author"
+					}
+				],
+				"date": "2013",
+				"DOI": "10.1016/j.gloenvcha.2013.05.005",
+				"ISSN": "0959-3780",
+				"abstractNote": "Previous estimates of the land area available for future cropland expansion relied on global-scale climate, soil and terrain data. They did not include a range of constraints and tradeoffs associated with land conversion. As a result, very large values of the global land reserve area have been estimated. Here, we define potentially available cropland (PAC) as the moderately to highly productive land that could be used in the coming years for rainfed farming, with low to moderate capital investments, and that is not forested, legally protected, or already intensively managed. This productive land is underutilized rather than unused as it has ecological or social functions. We also define potentially available cropland that accounts for trade-offs between gains in agricultural production and losses in ecosystem and social services from intensified agriculture (PACt), to include only the PAC that would entail low ecological and social costs with conversion to cropland. The objective of this study is to reconceptualize and then estimate the size and geographic distribution of PAC and PACt. In contrast to previous studies, we adopt a “bottom-up” approach by analyzing detailed, fine scale observations and expert knowledge for six countries or regions that are often assumed to include a large reserve of PAC. We conclude first that there is substantially less potential additional cropland than is generally assumed once constraints and trade offs are taken into account, and secondly that converting land is always associated with significant social and ecological costs. There are few remaining places with “free and easy” lands.",
+				"issue": "5",
+				"language": "ENG",
+				"libraryCatalog": "publications.jrc.ec.europa.eu",
+				"pages": "892-901",
+				"publicationTitle": "Global Environmental Change-Human and Policy Dimensions",
+				"volume": "23",
+				"attachments": [
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [
+					"JRC.H.5-Land Resources Management"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://publications.jrc.ec.europa.eu/repository/handle/JRC56837",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Eco-innovation: When Sustainability and Competitiveness Shake Hands",
+				"creators": [
+					{
+						"lastName": "Carrillo Hermosilla",
+						"firstName": "Javier",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Del Rio Gonzalez",
+						"firstName": "Pablo",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Könnölä",
```

Comment:
If you decide to go the RegExp route, would probably want to use XRegExp instead to cover all the umlauts, etc. This makes regexp approach somewhat less appealing, because you also have to consider that strings may not be normalized to NFC